### DISCUSSION TITLE
Contact authors

### DISCUSSION THREAD

**Discussion by seifbch** [dchristl/esp32_nat_router_extended#56] [General]
Est ce que je peux contacter l'un des développeur j'ai une petite fonctionnalité que je veux l'ajouter si c'est possible ?
Merci d'avance

**Comment by dchristl** [comment]
Hello seifbch,

you can contact me (I'm the only one) if you have a feature request. Please write in english language. Thank you.

Regards
Danny

**Reply by acc-git26** [reply]
Something's wrong when I try to upload the esp32s3nat_extended_full_v7.2.0 firmware to the esp32s3. It doesn't run.
Can you help me?

**Comment by dariuszce** [comment]
Hi, could you extend your program to support the ESP32-ETH01 board, so that the ESP receives an IP address from the LAN (or is manually set), and WiFi acts as an AP point as before?
